README.md

```diff
@@ -87,13 +88,15 @@ Follow these steps to deploy your first Pulumi program, using AWS Serverless Lam
 
 1. **Install**:
 
-    To install the latest Pulumi release, run the following (see full [installation instructions](https://pulumi.io/install) for additional installation options):
+    To install the latest Pulumi release, run the following (see full
+    [installation instructions](https://www.pulumi.com/docs/reference/install/) for additional installation options):
 
     ```bash
     $ curl -fsSL https://get.pulumi.com/ | sh
     ```
 
-2. **[Configure your Cloud Provider](https://pulumi.io/install#cloud-configuration)** so that Pulumi can deploy into it.
+2. **[Configure your Cloud Provider](https://www.pulumi.com/docs/reference/install/#cloud-configuration)**
```

Comment:
The `#cloud-configuration` portion of this link seems to be lost in the sands of time (this change happened even before the site migration). And we don't really have a single place to "Configure your Cloud Provider" because the instructions are all spread out across the clouds.

I wonder if we should delete this whole Get Started section of the docs, and just link to https://www.pulumi.com/docs/quickstart/

The upside is we'll be funneling folks to where we want them to go to get started, and the instructions for the various clouds, and platforms will all make sense. The downside is we don't show some of the simple pulumi commands in this doc, but that's probably fine.